### ISSUE TITLE
build for production

### ISSUE THREAD

**Issue by NogaB** [angular/material-start#161]
Hi,  it's more of a question than an isuue. I apologize if this is not the correct place. I want to upload to server the app I built based on your template. I use jspm bundle. Then i get errors about uaing undefuned builderHelper - the usage of i see in the transpiled bundled code.
Not sure how I should prepare the files for production or how to handle this specific problem. Thanks

**Comment by nh-99** [comment]
Hi there. Just did my first production install of my app based on this project.

The process was fairly straightforward. I ran `npm install -g jspm`, then ran `jspm bundle src/app` from the root directory of the project. At that point, I had a build.js file that contained all of my dependencies. In the index.html, I added `<script src="build.js"></script>` right after [this line](https://github.com/angular/material-start/blob/master/app/index.html#L51). Then I just installed nginx and copied the app directory over to my /var/www/html directory.
